book/src/external-resizer.md

```diff
@@ -0,0 +1,24 @@
+# CSI external-resizer
+
+## Status and Releases
+
+**Git Repository:** https://github.com/kubernetes-csi/external-resizer/
+
+**Status:** Alpha
+
+Latest release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
```

Comment:
Can you also add Min CSI Version and Max CSI Version columns like [this](https://github.com/kubernetes-csi/docs/pull/145/files#diff-6dea92996f4a334bc37787641b24ff90R9)